sdk/formrecognizer/Azure.AI.FormRecognizer/src/TrainingStatus.cs

```diff
@@ -6,21 +6,25 @@
 namespace Azure.AI.FormRecognizer.Training
 {
     /// <summary>
+    /// Describes the status of a training operation.
     /// </summary>
     [CodeGenModel("TrainStatus")]
 #pragma warning disable CA1717 // Only FlagsAttribute enums should have plural names
     public enum TrainingStatus
```

Comment:
I totally guessed the descriptions here. Let me know if I got something wrong.

Reply:
LGTM